recipes/wildmagic/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "WildMagic" %}
+{% set version = "5.14" %}
+{% set version_major = "5" %}
+{% set version_minor = "14" %}
+{% set sha256 = "951330bca2328d537dac2459bf154845295b8c4576a0adca5ffbf31b5a778d80" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}{{ version_major }}p{{ version_minor }}.zip
+  url: http://www.geometrictools.com/Downloads/WildMagic{{ version_major }}p{{ version_minor }}.zip
+  sha256: {{ sha256 }}
+
+  patches:
+    - wm5_path.patch
+
+build:
+  number: 0
+  skip: True  # [win]
+  binary_has_prefix_files:
+    - lib/libWm5Applications.so.5.14
+
+requirements:
+  build:
+    - requests
```

Comment:
Why do we need `requests` at build time? Also if we do need `requests`, why don't we also need `python`? Do we need `requests` at run time?